epuck_ros2_cpp/src/driver.cpp

```diff
@@ -443,6 +469,8 @@ class EPuckPublisher : public rclcpp::Node {
 
   float mWheelDistance;
   float mWheelRadius;
+
+  int mTofInitialized;
```

Comment:
Why not using a boolean here?

Reply:
The int value comes from:
https://github.com/cyberbotics/epuck_ros2/blob/f3296b4c669aabe43f8e22019062e347169cb043/epuck_ros2_cpp/src/driver.cpp#L104

I have just renamed to `mTofInitStatus`, maybe it makes more sense now?

Reply:
Yes indeed this is now much more clear.